yt/data_objects/construction_data_containers.py

```diff
@@ -1290,35 +1290,51 @@ class YTSmoothedCoveringGrid(YTCoveringGrid):
     _min_level = None
 
     @wraps(YTCoveringGrid.__init__)
-    def __init__(self, *args, **kwargs):
+    def __init__(self, *args, interp="linear", nbuf=None, **kwargs):
         ds = kwargs["ds"]
         self._base_dx = (
             ds.domain_right_edge - ds.domain_left_edge
         ) / ds.domain_dimensions.astype("float64")
         self.global_endindex = None
         YTCoveringGrid.__init__(self, *args, **kwargs)
         self._final_start_index = self.global_startindex
+        # Assign order (covergence) based on interpolation method
+        if interp == "linear":
+            order = 2
+        elif interp == "cubic":
+            order = 4
+        elif interp == "quintic":
+            order = 6
+        elif interp == "akima":
+            order = -1
+        elif interp == "natural":
+            order = -2
+        else:
+            raise NotImplementedError
```

Comment:
```suggestion
        order = {
            "natural": -2,
            "akima": -1,
            "linear": 2,
            "cubic": 4,
            "quintic": 6,
           }[interp]
```

Reply:
Btw I think `kind` would be a clearer name than "interp" as a user-exposed parameter.

Reply:
I agree. I will update that!